src/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -64,6 +64,26 @@ public override IEnumerable<TimeFunction> TimeFunctions(bool requiresRoundtrippi
                 ((testFile, time) => { testFile.LastWriteTimeUtc = time; }),
                 ((testFile) => testFile.LastWriteTimeUtc),
                 DateTimeKind.Utc);
+            yield return TimeFunction.Create(
+                ((testFile, time) => CopytoOperation()),
+                ((testFile) => testFile.LastWriteTimeUtc),
```

Comment:
I don't understand how this fits in with this TimeFunction.Create.  The purpose of that helper as I understand it is to modify testFile in the first delegate and then validate with the second that it was appropriately changed.  But in this new case, testFile isn't being modified in the original delegate, and thus the second delegate is irrelevant.  Why is this new test then plugging into this in this way?

Reply:
I wanted to use the same behavior if we wanted to add some other similar tests in the future(Checking that the file still has millisecond attribute after some operation).

But i was not able to properly utilize existing this. For time being, i will just move this to its own test.